### ISSUE TITLE
Adgurad crashes browser on specific site with Russian filter on

### ISSUE THREAD

**Issue by AjvarXX** [AdguardTeam/AdguardBrowserExtension#1671]
### Issue Details

Waterfox Classic suddenly crashes on either ukr.net or when I click there a link to my inbox mail.ukr.net. Note that if I uploaded page of my mail and then turned on adguard beta extension I can reload my mail. But if I click that link to mail on ukr.net again - it crashes.
Profile is fresh - a day with synced history and just a little bunch of freshly installed extensions which don't interact as crash happens even if I disable all of them except aduard beta.

On another instance of freshly installed browser with synced history on another laptop I get total crash just trying to launch ukr.net page.

Form what I can see it happens with earlier versions of beta Adguard extension but not with non-beta from mozilla addon page.

- **AdGuard version:**
  - AdGuard v3.5.5
- **Browser and version:**
  - Waterfox Classic (fork of Firefox ESR)
- **Operating system and version:**
  - Windows 7
- **Filters you use in AdGuard:**
  -   - At first I thought it crashes even on Basic but then... I found that ONLY Language-Specific Adguard Russian Filter is the one giving this crash.
- **Stealth Mode settings:**
  - Disabled
- **Other browser extensions:**
  - Non-relevant as iI disabled those but here: Always on Top	2.1.25.1, Browsec VPN - Free and Unlimited VPN	3.27.6, Form History Control (II)	2.3.0.0, Google Keep Notes	65.0.0, Hide Caption Titlebar Plus ⁴, In My Pocket	0.11.5, New Tab Tools	89.1, OmniSidebar	1.6.15, Session Manager	0.8.1.13, Stylus	1.5.11, Tab Groups

### Expected Behavior

No crash
### Actual Behavior

Browser crashes
### Screenshots

<details><summary>Screenshot:</summary>

</details>

### Additional Information
Disabling multiprocessing did not help.

**Comment by adgkart** [comment]
Could you please export settings from AdGuard AdBlocker (AdGuard Settings - General - Export settings) and send them to us?

**Comment by AjvarXX** [comment]
Of course. Earlier is with working settings. Then I switched Russian filter on and tried that site (go to Inbox -button "Received" - from ukr.net). On a 2nd try it crashed.
[Settings.zip](https://github.com/AdguardTeam/AdguardBrowserExtension/files/4994625/Settings.zip)
